Orange/widgets/unsupervised/owhierarchicalclustering.py

```diff
@@ -1092,10 +1099,12 @@ def commit(self):
 
         if not selected_indices:
             self.send("Selected Data", None)
-            self.send("Other Data", None)
+            annotated_data = create_annotated_table(items, selected_indices) \
```

Comment:
Not really an issue, but since you're going to make another commit anyway, can you replace `selected_indices` with `[]`, so that it's obvious this call will always select all (or no) instances.